doc/user/content/get-started/_index.md

```diff
@@ -60,13 +60,21 @@ this exact purpose.
 
 ### Standard SQL support
 
-Materialize follows the SQL standard (SQL-92) implementation, so you interact
-with it like any relational database: using SQL. You can build complex
-analytical workloads using **[any type of join](/sql/select/join/)** (including
-non-windowed joins and joins on arbitrary conditions), but you can also
-leverage exciting new SQL patterns enabled by streaming like
-[**Change Data Capture (CDC)**](/integrations/#databases),
-[**temporal filters**](/sql/patterns/temporal-filters/), and [**subscriptions**](/sql/subscribe/).
+You can interact with Materialize using SQL. You can build complex analytical
```

Comment:
```suggestion
The language used to interact with Materialize is **SQL**. You can build complex
analytical
```

Reply:
"You can" makes it sound like you can use SQL, but you can also use something else.

Reply:
How about "You interact with Materialize using SQL"?  "The language used to interact ... is SQL" seems slightly awkward.